### PR TITLE
update `specifics/plug.md`

### PR DESCRIPTION
**Pull request by Nao000**
this lessons will be error if no update.

and this is not only ja.

### CODE REVIEW

**Comment by doomspork** [comment]
@Nao000 do you have an example of this failing?  This change does not look correct 🤔

**Comment by Nao000** [comment]
I'm sorry. this typo is only japanese mode. 

[english]
> To start let’s create a file at **lib/example/router.ex** and copy the following into it:
> 
> defmodule **Example.Router** do
>   use Plug.Router
>   ...

and 

> Back in lib/example.ex, we need to add **Example.Router** into the web server supervisor tree. Swap out the Example.HelloWorldPlug plug with the new router:
> 
> def start(_type, _args) do
>   children = [
>     Plug.Adapters.Cowboy.child_spec(:http, **Example.Router,** [], port: 8080)
>   ]

https://elixirschool.com/en/lessons/specifics/plug/#plugrouter

[japanese  same part]
> 手始めに、**lib/plug/router.ex**というファイルを作り、以下をコピーしましょう:
> 
> defmodule **Example.Plug.Router** do
>   use Plug.Router

and

> lib/example.exにもどり、**Example.Router**をWebサーバーの管理下に追加してください。そして、Example.HelloWorldPlugPlugを新しいルータに退避させてください
> 
> def start(_type, _args) do
>   children = [
>     Plug.Adapters.Cowboy.child_spec(:http, **Example.Router**, [], port: 8080)
>   ]
https://elixirschool.com/ja/lessons/specifics/plug/#plugrouter%E3%81%AE%E4%BD%BF%E7%94%A8

and This error occured.
```
D:*****\elixir\plug101\example>mix run --no-halt
Compiling 1 file (.ex)

20:45:26.573 [info]  Application example exited: exited in: Example.start(:normal, [])
    ** (EXIT) an exception was raised:
        ** (UndefinedFunctionError) function Example.Router.init/1 is undefined (module Example.Router is not available)
            Example.Router.init([])
            (plug) lib/plug/adapters/cowboy.ex:191: Plug.Adapters.Cowboy.dispatch_for/2
            (plug) lib/plug/adapters/cowboy.ex:48: Plug.Adapters.Cowboy.args/4
            (plug) lib/plug/adapters/cowboy.ex:137: Plug.Adapters.Cowboy.child_spec/4
            (example) lib/example.ex:9: Example.start/2
            (kernel) application_master.erl:273: :application_master.start_it_old/4
** (Mix) Could not start application example: exited in: Example.start(:normal, [])
    ** (EXIT) an exception was raised:
        ** (UndefinedFunctionError) function Example.Router.init/1 is undefined (module Example.Router is not available)
            Example.Router.init([])
            (plug) lib/plug/adapters/cowboy.ex:191: Plug.Adapters.Cowboy.dispatch_for/2
            (plug) lib/plug/adapters/cowboy.ex:48: Plug.Adapters.Cowboy.args/4
            (plug) lib/plug/adapters/cowboy.ex:137: Plug.Adapters.Cowboy.child_spec/4
            (example) lib/example.ex:9: Example.start/2
            (kernel) application_master.erl:273: :application_master.start_it_old/4
```

File: ja/lessons/specifics/plug.md [hunk 1]
```diff
@@ -141,12 +141,12 @@ end
 
 これは必要最小限のルータですが、コード自身がうまく中身を説明してくれているはずです。`use Plug.Router`でマクロをいくつか読み込み、それから2つの組み込みのPlug、`:match`と`:dispatch`を配置します。2つのルータが定義され、1つはルート(`/`)へのGETリクエストを制御します。2つ目ではそれ以外の全てのリクエストにマッチして、404メッセージを返すことができます。
 
-`lib/example.ex`にもどり、`Example.Router`をWebサーバーの管理下に追加してください。そして、`Example.HelloWorldPlug`Plugを新しいルータに退避させてください
+`lib/example.ex`にもどり、`Example.Plug.Router`をWebサーバーの管理下に追加してください。そして、`Example.HelloWorldPlug`Plugを新しいルータに退避させてください
```
Comment (riseshia) [inline ja/lessons/specifics/plug.md]: https://github.com/elixirschool/elixirschool/blame/a014ce3e2ea74a8a7b612095a8f0f6c28325b908/en/lessons/specifics/plug.md#L156

File: ja/lessons/specifics/plug.md [hunk 2]
```diff
@@ -141,12 +141,12 @@ end
...
 
 ```elixir
 def start(_type, _args) do
   children = [
-    Plug.Adapters.Cowboy.child_spec(:http, Example.Router, [], port: 8080)
+    Plug.Adapters.Cowboy.child_spec(:http, Example.Plug.Router, [], port: 8080)
```
Comment (riseshia) [inline ja/lessons/specifics/plug.md]: https://github.com/elixirschool/elixirschool/blame/a014ce3e2ea74a8a7b612095a8f0f6c28325b908/en/lessons/specifics/plug.md#L162

Comment (Nao000) [reply ja/lessons/specifics/plug.md]: I'm sorry, I have misunderstood. ..

**Pull request closed without merging**